conda/README.md

```diff
@@ -0,0 +1,192 @@
+# Conda for Package Managment of Zowe Plugins
+
+As Zowe is composed of components which can be extended by plugins, 
+A standardized and simple way to find, install, upgrade, and list plugins in your Zowe environment
+Is important to make it easy to get the most out of Zowe.
+
+Package management as a concept generally provides a way to find packages such as plugins,
+Check and possible co-install dependencies the package has, and ultimately install the desired package.
+Post-install, management tasks such as upgrading and uninstalling are common.
+
+Conda is one such package manager, and if you are familiar with apt, yum, or npm, you will find
+That using conda is very similar. But, there are some important abilities that make conda stand out:
```

Comment:
*that using Conda is very similar. But, there are some important abilities that make Conda stand out: